src/index.html

```diff
@@ -2,22 +2,90 @@
 <html>
 <head>
     <title>ORCs Project</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" href="Stil.css">
+
+
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
 </head>
 <body>
-<h1>Online Role-playing Characters</h1>
+
+
 
 <h2>
     <small>Project status:</small>
     undeployed
 </h2>
 
 <h3>Description</h3>
-<p>The application is going to support D&D as first role-playing game. Make Character creation and management more
-    easier is the main goal. Character creation by the free roles of the game. Character management by users. Share
-    character sheets.</p>
 
 <p><a href="https://github.com/LunaWorks/Orcs-Web">Repository</a></p>
 
 <p>Developer: <a href="https://github.com/LunaWorks">LunaWorks</a></p>
+
+<nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <ul class="nav navbar-nav">
+
+                <div class="navbar-brand">
+                    ORCs - Online Role-playing Characters
+                </div>
+            </ul>
+        </div>
+        <div class="collapse navbar-collapse" id="myNavbar">
+            <ul class="nav navbar-nav">
+                <li><a href="#">Home</a></li>
+                <li><a href="#"> <button>Characters</button></a></li>
```

Comment:
Bootstrap uses anchors here, not buttons!
In addition your button is not working!
Define <a href="characters.html">Characters</a> instead!